org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/Painter.java

```diff
@@ -175,25 +175,57 @@ public String getColor() {
   }
 
   /** @return True if facing North */
-  public boolean facingNorth() {
+  public boolean isFacingNorth() {
     return this.direction.isNorth();
   }
 
   /** @return True if facing East */
-  public boolean facingEast() {
+  public boolean isFacingEast() {
     return this.direction.isEast();
   }
 
   /** @return True if facing South */
-  public boolean facingSouth() {
+  public boolean isFacingSouth() {
     return this.direction.isSouth();
   }
 
   /** @return True if facing West */
-  public boolean facingWest() {
+  public boolean isFacingWest() {
     return this.direction.isWest();
   }
 
+  /**
+   * @deprecated use {@link Painter#isFacingNorth()}
```

Comment:
Does a deprecation warning show up in Javalab? I can't remember if we turned off all deprecation warnings or just for certain classes

Reply:
I think we intentionally hide deprecated warnings to avoid confusing students. However, if Javalab docs are still being updated from the source here, then this deprecation warning should at least show up in the docs.